### PR TITLE
Nerfs - Alpha labs

### PR DESCRIPTION
**Pull request by mikelgg93**
This PR brings nerf tutorial/test to the alpha labs content.

Additionally, all broken links are fixed, and a note on how to cite PI and Neon pipeline versions is added.

Neon tag for alpha lab content added to dynamic-rim

Todos:
- [x] Update Youtube link
- [x] Publish pyflux

### CODE REVIEW

File: src/alpha-lab/nerfs.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+But nowadays, thanks to recent advances in deep learning, we have an easier way to reconstruct and build 3D environments. Isn't that exciting?!
+
+## What are NeRFs and how do they work?
+
+That advance we are talking about are [Neural Radiance Fields](https://arxiv.org/pdf/2003.08934.pdf) or NeRFs 🔫. NeRFs are a relatively novel method that use deep neural networks. They learn how light and colour vary based on the viewer's location and direction. So, by providing this tool with a set of images of a scene from different angles, it can generate novel views that were never actually captured by the camera.
+
+With this technique, we can create high-fidelity and photorealistic 3D models that can be used for various applications such as virtual reality, robotics, urban mapping, or in our case, to understand how the wearer was moving and looking in their environment. This approach doesn't need endless pictures of the environment, just a set of frames and camera poses (where the camera was located and where it pointed to).
```
Comment (euryalus) [inline src/alpha-lab/nerfs.md:31]: What's the difference between "endless pictures of the environment" to "just a set of frames and camera poses"? It seems in a sense, the second is more (as it includes poses).

File: src/alpha-lab/nerfs.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+## That sounds cool! How we get those?
+
+Once you made a recording, we do not know where the camera was on each frame, and this is crucial information that we need to train the NeRF. No worries, COLMAP to the rescue! You can think of COLMAP as a puzzle solver. It takes your frames and figures out where the camera was located and where it was pointing. Something similar is used within our Reference Image Mapper. In fact, we use the poses this enrichment produces, and transform them to something nerfstudio can understand.
```
Comment (euryalus) [inline src/alpha-lab/nerfs.md:35]: " we do not know where the camera was on each frame" -> To me this sounds a bit like the camera was to be found on the frame. Maybe change the "on" to "for" or "during the recording of"?

File: src/alpha-lab/nerfs.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+## Now, what is NeRFStudio?
+
+[Nerfstudio](https://docs.nerf.studio/en/latest/) 🚜 is an open-source package that allows users to interactively create, visualise and edit NeRFs, and bundles several tools including a way to generate 3D meshes from the NeRF.
+
+Under the hood, NerfStudio is built on top of PyTorch and PyQt, and uses OpenGL for real-time rendering. It leverages the NeRF codebase to load and manipulate the models, and provides a high-level interface to interact with them. NerfStudio is still in active development, and new features and improvements are being added regularly.
```
Comment (euryalus) [inline src/alpha-lab/nerfs.md:41]: Out of curiosity, where is Nerfstudio using OpenGL?

File: src/alpha-lab/nerfs.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+This is not gonna be an easy path... 
+
+<details>
+    <summary>But if you insist...</summary><br>
+<!-- This is collapsed   -->
+    
+### What you'll need
+
+- A powerful computer with CUDA support (e.g. an Nvidia GPU) is a **must** for this to work
```
Comment (euryalus) [inline src/alpha-lab/nerfs.md:53]: I'd add here that the computer needs to support modern OpenGL. Which sadly excludes MACs.

**Review by euryalus — COMMENTED** [review]
Good stuff! I have left some comments on nerfs.md

File: src/alpha-lab/nerfs.md (hunk 3 above)
Comment (mikelgg93) [reply src/alpha-lab/nerfs.md:41]: My mistake, I thought the viewer was webGL, I see is ThreeJS and ReactJS though

**Comment by mikelgg93** [comment]
Ready to merge!

**Pull request merged**